tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -131,18 +167,39 @@ void GitCloneAndBuild(ExplorationTestDescription testDescription)
         );
     }
 
+    void DeleteExistingDirectory(string path)
+    {
+        var dirInfo = new DirectoryInfo(path);
+        SetAttributesNormal(dirInfo);
+        dirInfo.Delete(true);
+        return;
+
+        void SetAttributesNormal(DirectoryInfo dir)
+        {
+            foreach (var subDir in dir.GetDirectories())
+            {
+                SetAttributesNormal(subDir);
+                subDir.Attributes = FileAttributes.Normal;
+            }
+            foreach (var file in dir.GetFiles())
+            {
+                file.Attributes = FileAttributes.Normal;
+            }
+        }
+    }
+
```

Comment:
nit: I don't _think_ this is used anywhere AFAICT? 🤔 
```suggestion
```

Reply:
Fix in 9f23fdedaa2a7645d2ae2d2edbaf79936c9e57ef